styles/instagram/catppuccin.user.css

```diff
@@ -654,32 +654,59 @@
     #catppuccin(@darkFlavor,
         @accentColor);
```

Comment:
This should be `#catppuccin(@lookup, @accent)` instead of `#catppuccin(@darkFlavor, @accentColor)` I think.

Reply:
sure, i'll change it